[2025-05-25T07:10:36.716+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2025-05-25T07:10:37.540+0000] {taskinstance.py:2613} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: load_file_to_psql.generate_file scheduled__2025-04-23T00:00:00+00:00 [queued]>
[2025-05-25T07:10:38.006+0000] {taskinstance.py:2613} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: load_file_to_psql.generate_file scheduled__2025-04-23T00:00:00+00:00 [queued]>
[2025-05-25T07:10:38.090+0000] {taskinstance.py:2866} INFO - Starting attempt 1 of 1
[2025-05-25T07:10:39.089+0000] {taskinstance.py:2889} INFO - Executing <Task(PythonOperator): generate_file> on 2025-04-23 00:00:00+00:00
[2025-05-25T07:10:39.586+0000] {warnings.py:112} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:70: DeprecationWarning: This process (pid=1631) is multi-threaded, use of fork() may lead to deadlocks in the child.
  pid = os.fork()

[2025-05-25T07:10:39.682+0000] {standard_task_runner.py:72} INFO - Started process 1693 to run task
[2025-05-25T07:10:39.886+0000] {standard_task_runner.py:104} INFO - Running: ['***', 'tasks', 'run', 'load_file_to_psql', 'generate_file', 'scheduled__2025-04-23T00:00:00+00:00', '--job-id', '236', '--raw', '--subdir', 'DAGS_FOLDER/data_quality.py', '--cfg-path', '/tmp/tmpqbi6autr']
[2025-05-25T07:10:39.968+0000] {standard_task_runner.py:105} INFO - Job 236: Subtask generate_file
[2025-05-25T07:10:42.716+0000] {task_command.py:467} INFO - Running <TaskInstance: load_file_to_psql.generate_file scheduled__2025-04-23T00:00:00+00:00 [running]> on host b4cf172ed0b9
[2025-05-25T07:10:49.568+0000] {taskinstance.py:3132} INFO - Exporting env vars: AIRFLOW_CTX_DAG_EMAIL='https://t.me/Shust_DE' AIRFLOW_CTX_DAG_OWNER='@Shust_DE' AIRFLOW_CTX_DAG_ID='load_file_to_psql' AIRFLOW_CTX_TASK_ID='generate_file' AIRFLOW_CTX_EXECUTION_DATE='2025-04-23T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2025-04-23T00:00:00+00:00'
[2025-05-25T07:10:49.621+0000] {taskinstance.py:731} INFO - ::endgroup::
[2025-05-25T07:10:52.447+0000] {python.py:240} INFO - Done. Returned value was: None
[2025-05-25T07:10:52.820+0000] {taskinstance.py:340} INFO - ::group::Post task execution logs
[2025-05-25T07:10:52.851+0000] {taskinstance.py:352} INFO - Marking task as SUCCESS. dag_id=load_file_to_psql, task_id=generate_file, run_id=scheduled__2025-04-23T00:00:00+00:00, execution_date=20250423T000000, start_date=20250525T071037, end_date=20250525T071052
[2025-05-25T07:10:53.760+0000] {local_task_job_runner.py:266} INFO - Task exited with return code 0
[2025-05-25T07:10:53.821+0000] {local_task_job_runner.py:245} INFO - ::endgroup::
